### ISSUE TITLE
Unknown column type "json_array" requested

### ISSUE THREAD

**Issue by hgiesenow** [sulu/sulu#6284]
| Q | A
| --- | ---
| Sulu Version | 2.3.5
| PHP Version | 7.4
| DB Version | mariadb-10.3
| Browser Version | -

# Actual Behavior

After upgrading with `composer update`, the command `bin/adminconsole sulu:build dev` fails with the message
```
Target: database

    doctrine:schema:update  ({"--force":true})
    
        
        In Exception.php line 125:
                                                                                                                                                                                                                                               
          Unknown column type "json_array" requested. Any Doctrine type that you use has to be registered with \Doctrine\DBAL\Types\Type::addType(). You can get a list of all the known types with \Doctrine\DBAL\Types\Type::getTypesMap().  
           If this error occurs during database introspection then you might have forgotten to register all database types for a Doctrine Type. Use AbstractPlatform#registerDoctrineTypeMapping() or have your custom types implement Type#g  
          etMappedDatabaseTypes(). If the type name is empty you might have a problem with the cache or forgot some mapping information.
```

# Expected Behavior

The database is being created correctly

# Steps to Reproduce

```
composer update
bin/adminconsole sulu:build dev
```

# Possible Solutions

This error is likely caused by `doctrine/dbal:3.1.3`. When sticking to 2.6 (`composer require doctrine/dbal:"^2.6"`) it works.
A solution would be to adjust the constraints to `^2.6` instead of `^2.6|^3.0` until the deprecated "json_array" type is removed completely from sulu.

**Comment by alexander-schranz** [comment]
Thx for report. This is fixed in #6271 and will be part of the next release. You can downgrade dbal at current state or use `~2.3.5@dev` until we release 2.3.6 version.

**Comment by alexander-schranz** [comment]
See also: https://github.com/doctrine/orm/issues/9078 so manual database update is required here.

**Comment by niklasnatter** [comment]
Thanks for reaching out! For completion, this is caused by the newly released `doctrine/orm@2.10.0` and`doctrine/dbal@3.1.3` packages. We have already fixed the problem in #6271 which will be included in the `Sulu 2.3.6` release.

Until `Sulu 2.3.6` is released, you could downgrade the `doctrine/dbal` package to version 2 with the following command:
```
composer require doctrine/dbal:"^2.6"
```
